src/components/Masthead.js

```diff
@@ -48,6 +52,7 @@ type MastheadDefaultProps = {
   logoUri: string | null;
   logoAlt: string | null;
   homeRoute: string | null;
+  homeUrl: ?string;
```

Comment:
oops, I'll make them match.

One thought - the `string | null` typing is a bit odd. I won't break away from that here as a one off, but if a prop is not required, I would not expect to define it and pass null, I would just omit it, `null` and `undefined` being different. Since proptypes and flow define required in different ways, I would expect to see in proptypes: `myProp: string;` and then in the flow type: `myProp: ?string` to use the [maybe type](https://flow.org/en/docs/types/primitives/#toc-maybe-types). I can always pass an empty string if need be, like if I'm defining a prop, but maybe it's dependent on other data being available.

(What I did is still mismatched and needs to be corrected.)